themes/bootstrap5/js/common.js

```diff
@@ -284,9 +304,9 @@ var VuFind = (function VuFind() {
 
   /**
    * Return a spinner html element with loading text
-   * @param {string|null} text [Optional] Translation key to append inside span wrapper, default loading_ellipsis
-   * @param {string} extraClass [Optional] Extra class string to add for spinner wrapper
-   * @returns {HTMLSpanElement}
+   * @param {string|null} [text]       Translation key to append inside span wrapper (default = null)
```

Comment:
Maybe we should clarify how defaults work here:

```suggestion
   * @param {string|null} [text]       Translation key to append inside span wrapper; loading_ellipsis is used if null (default = null)
```